lib/vanagon/common/pathname.rb

```diff
@@ -28,15 +28,18 @@ class Pathname
       # @param [String, Integer] group the groupname or GID to use when this file is archived
       # @param [Boolean] config mark this file as a configuration file, stored as private state
       #   and exposed through the {#configfile?} method.
+      # @param [string] id the id to pass to wix objects during directory generation
       # @return [Vanagon::Common::Pathname] Returns a new Pathname instance.
-      def initialize(path, mode: nil, owner: nil, group: nil, config: false)
+      def initialize(path, mode: nil, owner: nil, group: nil, config: false, id: nil)
```

Comment:
Right, which is likely what i would do, but now to me rubocop is making things less clear yes? seems counterproductive to what rubocop is for.